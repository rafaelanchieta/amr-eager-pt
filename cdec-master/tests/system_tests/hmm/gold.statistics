-lm_nodes 10
-lm_edges 40
-lm_paths 16807
+lm_nodes 64
+lm_edges 238
+lm_paths 16807
+lm_trans it is quite understandable .
constr_nodes 64
constr_edges 238
constr_paths 16807
