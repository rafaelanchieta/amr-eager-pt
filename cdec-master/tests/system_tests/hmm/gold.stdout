ce est tout à fait compréhensible . ||| it is quite understandable . ||| 0-0 1-1 5-2 5-3 6-4
